core/src/main/java/google/registry/flows/domain/DomainRenewFlow.java

```diff
@@ -328,7 +328,7 @@ private void verifyRenewAllowed(
       checkHasBillingAccount(registrarId, existingDomain.getTld());
     }
     verifyUnitIsYears(command.getPeriod());
-    // We only allow __REMOVE_PACKAGE__ token on bulk pricing domains for now
+    // We only allow __REMOVEDOMAIN__ token on bulk pricing domains for now
```

Comment:
I'm gathering that this comment change is orthogonal, but I don't think `__REMOVEDOMAIN__` conveys the same semantic meaning as `__REMOVE_PACKAGE__` did (also due to lack of underscore between words it's less readable).  When you're looking at a domain update EPP with a token that merely says "remove domain", it's not really clear what that actually means.  It kind of sounds like it should be a domain delete.  A domain update EPP that says "remove package" on the other hand makes sense; it's saying to remove the domain from a package.  So the new name of the token should be something that incorporates the words "bulk pricing" so it's clear what it's actually doing.